knowage-vue/src/modules/documentExecution/dashboard/Dashboard.d.ts

```diff
@@ -41,8 +37,9 @@ export interface IWidget {
     dataset: number | null
     type: string
     columns: IWidgetColumn[]
-    settings: ITableWidgetSettings | ISelectionsWidgetSettings | ISelectorWidgetSettings | IHTMLWidgetSettings | ITextWidgetSettings | IHighchartsWidgetSettings | IChartJSWidgetSettings | IImageWidgetSettings | ICustomChartWidgetSettings
+    settings: any
```

Comment:
Linter is giving us errors on a lot of places because we didn't use casting anywhere and it doesn't know what kind of widget settings will it be (even though we know for example in some **chart** component it has to be **IHighchartsSettings** etc.). We can return it, but it will show errors, but I don't think they are the **blocking** ones.

Reply:
OK. We need to figure out in the future how to keep types and ESLint together avoiding errors.